appium/tests/screenobjects/inbox.screen.ts

```diff
@@ -28,10 +28,12 @@ class InboxScreen extends BaseScreen {
     await $(`~${email}`).click();
   }
 
-  clickOnEmailBySubject = async (subject: string) => {
+  clickOnEmailBySubject = async (subject: string, withScroll: boolean = true) => {
     const selector = `~${subject}`;
-    if (await (await $(selector)).isDisplayed() !== true) {
-      await TouchHelper.scrollDown();
+    if (withScroll) {
+      if (await (await $(selector)).isDisplayed() !== true) {
+        await TouchHelper.scrollDown();
+      }
     }
     await ElementHelper.waitAndClick(await $(selector), 500);
   }
```

Comment:
This is a mess (not your fault), should be addressed here https://github.com/FlowCrypt/flowcrypt-ios/issues/1071

Was the scroll causing you trouble? It's supposed to only scroll when the email is not present, but it's sloppily written - it's testing too fast. Maybe it wasn't loaded yet. Try, instead of this `withScroll`, to add a 2000ms wait before you call `clickOnEmailBySubject`.